homeassistant/components/airthings_ble/sensor.py

```diff
@@ -189,4 +190,12 @@ def __init__(
     @property
     def native_value(self) -> StateType:
         """Return the value reported by the sensor."""
-        return self.coordinator.data.sensors[self.entity_description.key]
+        _LOGGER.debug(
+            "updating %s, data: %s", self.entity_description.key, self.coordinator.data
```

Comment:
Do you need the debug statement in production code ?

Reply:
I can probably drop it, but wouldn't this be good in the event of enabling debug logging in production?

Reply:
It will create a log entry at your update interval, which at least by default is every 30 seconds, so you'll get 120*24 log entries per day.  

Also your data is already being recorded each time the entity is updated (filling in your graph) so I don't see how this helps.

Reply:
Makes sense, dropped.